CSharpFunctionalExtensions/Result/Dto/HttpResponseMessageJsonExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿#nullable enable
+using System.Text.Json;
+using System.Threading.Tasks;
+
+using CSharpFunctionalExtensions;
+
+namespace System.Net.Http.Json
```

Comment:
Extension methods that are not used in their context are hard to discover.
I am using the same namespace as System.Net.Http.Json.HttpContentJsonExtensions that exposes ReadFromJsonAsync.
In my opinion, it will make this call much more visible.
If not, I think we should move it the CSharpFunctionalExtensions namespace so this method will be exposed to users of Result<>.
What do you think?

Reply:
Every modern IDE currently provides rich intelligence. Even VS without JB tools/extensions installed may prompt you to import namespaces that are not imported, and also show extensions from namespaces that are not imported.

I think that it makes much easier to do code review and to read a code which has exmplicit `using`s for all used namespaces.

Serializing/deserializing `Result`s to json is a very niche feature in my opinion. And it should not be visible to default, users should explicitely ask for it (by importing a corresponding namespace)